nessie/src/main/java/org/apache/iceberg/nessie/NessieUtil.java

```diff
@@ -89,4 +90,15 @@ private static String commitAuthor(Map<String, String> catalogOptions) {
     return Optional.ofNullable(catalogOptions.get(CatalogProperties.USER))
         .orElseGet(() -> System.getProperty("user.name"));
   }
+
+  /**
+   * Indicates whether the Nessie Catalog should automatically create namespaces implied by table
+   * identifiers.
+   *
+   * @param catalogOptions The options where to look for configuration settings.
+   */
+  static boolean shouldCreateImpliedNamespaces(Map<String, String> catalogOptions) {
+    String create = catalogOptions.get(CREATE_IMPLIED_NAMESPACES);
```

Comment:
Maybe the property name has to be reversed (because it is enabled by default). Like `DISBALE_IMPLIED_NAMESPACE_CREATION`

Reply:
fixed